src/main/webapp/app/entities/notification.model.ts

```diff
@@ -22,3 +22,61 @@ export class Notification implements BaseEntity {
         this.notificationType = notificationType;
     }
 }
+
+/**
+ * Corresponds to the NotificationTitleTypeConstants(.java) constant Strings in the server
```

Comment:
nit: Corresponds to the server-side NotificationTitleTypeConstants(.java) constant Strings